config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": null,
+    "password": "$abaidee11",
```

Comment:
We prob don't want to push your pw do we?

Reply:
updated! Thanks!